src/components/User-Form/Update-Form.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import { CountryDropdown, RegionDropdown } from 'react-country-region-selector'
+
+const UpdateUserInfoForm = ({address, userAbout, updateHandler, back}) => {
+
+  const [about, setAbout] = useState(userAbout)
+  const [userAddress, setAddress] = useState(address)
+
+  const inputHandler = (e) => {
+    switch(e.target.id) {
+      case 'street':
+        setAddress({...userAddress, street: e.target.value})
+        break;
+      case 'city':
+        setAddress({...userAddress, city: e.target.value})
+        break;
+      case 'state':
+        setAddress({...userAddress, state: e.target.value})
+        break;
+      case 'zip':
+        setAddress({...userAddress, zip: e.target.value})
+        break;
+      case 'country':
+        setAddress({...userAddress, country: e.target.value})
+        break;
+      case 'about':
+        setAbout(e.target.value)
+        break;
+      default:
+        return userAddress
+    }
+  }
+
+  const submitHandler = (event) => {
+    event.preventDefault()
+    updateHandler(userAddress, about)
+  }
+
+  return (
+    <div className='form-wrapper'>
+        <form onChange={inputHandler} onSubmit={submitHandler}>
+          <div>
+            <label htmlFor='country'>Country:</label>
+            <CountryDropdown value={userAddress.country} id='country'  />
+            <label htmlFor='state'>State/Region:</label>
+            <RegionDropdown country={userAddress.country} value={userAddress.state} id='state'  />
```

Comment:
I think for MVP, it would be best to set the country as the united states due to the different address formatting.